.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: [push]
```

Comment:
Maybe we could also run this before a merge? Or is that implied because it runs on every push?

Reply:
Merge commits are pushed, so yes this runs then as well. It just runs on each time commit(s) is/are pushed. However, it then runs after the merge being pushed. Doing it before requires bors or something similar. Do you think we need that here?

Reply:
I don't think we need this. Still, it would be nice to have.

Reply:
Let's leave it out for now. If we have a need for it later, we can always add it